src/embedded.ts

```diff
@@ -147,6 +149,26 @@ export class RoomWidgetClient extends MatrixClient {
     ) {
         super(opts);
 
+        const transportSend = this.widgetApi.transport.send.bind(this.widgetApi.transport);
+        // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Why?

Reply:
This was to avoid having to copy over the type hint of the original function, but a hint might be needed after all since this is a generic function...